baselines/niid_bench/run_all.sh

```diff
@@ -0,0 +1,136 @@
+seed=$1
+
+# iid cifar10
+python -m niid_bench.main_fedavg partitioning=iid dataset_seed=$seed &
+python -m niid_bench.main_scaffold partitioning=iid dataset_seed=$seed &
+python -m niid_bench.main_fedprox partitioning=iid dataset_seed=$seed &
+python -m niid_bench.main_fedprox partitioning=iid mu=0.1 dataset_seed=$seed &
+python -m niid_bench.main_fedprox partitioning=iid mu=0.001 dataset_seed=$seed &
+python -m niid_bench.main_fedprox partitioning=iid mu=1.0 dataset_seed=$seed &
+python -m niid_bench.main_fednova partitioning=iid dataset_seed=$seed;
+
+# iid mnist
+python -m niid_bench.main_fedavg partitioning=iid dataset_seed=$seed dataset_name=mnist  &
+python -m niid_bench.main_scaffold partitioning=iid dataset_seed=$seed dataset_name=mnist  &
+python -m niid_bench.main_fedprox partitioning=iid dataset_seed=$seed dataset_name=mnist  &
+python -m niid_bench.main_fedprox partitioning=iid mu=0.1 dataset_seed=$seed dataset_name=mnist  &
+python -m niid_bench.main_fedprox partitioning=iid mu=0.001 dataset_seed=$seed dataset_name=mnist  &
+python -m niid_bench.main_fedprox partitioning=iid mu=1.0 dataset_seed=$seed dataset_name=mnist  &
+python -m niid_bench.main_fednova partitioning=iid dataset_seed=$seed dataset_name=mnist ;
+
+# iid fmnist
+python -m niid_bench.main_fedavg partitioning=iid dataset_seed=$seed dataset_name=fmnist  &
+python -m niid_bench.main_scaffold partitioning=iid dataset_seed=$seed dataset_name=fmnist  &
+python -m niid_bench.main_fedprox partitioning=iid dataset_seed=$seed dataset_name=fmnist  &
+python -m niid_bench.main_fedprox partitioning=iid mu=0.1 dataset_seed=$seed dataset_name=fmnist  &
+python -m niid_bench.main_fedprox partitioning=iid mu=0.001 dataset_seed=$seed dataset_name=fmnist  &
+python -m niid_bench.main_fedprox partitioning=iid mu=1.0 dataset_seed=$seed dataset_name=fmnist  &
```

Comment:
Are these being used in the table you show in the readme? (this comment applies to all `fedprox` experiments where you manually set the proximal term `mu`